### PR TITLE
i18n(zh-cn): Translate `start/migrate/from-tauri-2-beta.mdx`

### PR DESCRIPTION
**Pull request by GuoJikun**
#### Description

- What does this PR change? Give us a brief description. 
- Closes #

### CODE REVIEW

File: src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+title: 从 Tauri 2.0 Beta 升级
+i18nReady: false
+sidebar:
+  order: 16
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+本指南将引导您将 Tauri2.0 beta 应用程序升级到 Tauri2.0 rc 版本。
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx]:
```suggestion
本指南将引导您将 Tauri 2.0 Beta 应用程序升级到 Tauri 2.0 RC 版本。
```
Formatting nitpick.

File: src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    pnpm tauri migrate"
+  cargo='cargo install tauri-cli --version "^2.0.0-rc"
+    cargo tauri migrate'
+/>
+
+有关 `migrate` 命令的更多信息，请参阅[命令行接口参考](/reference/cli/#migrate)
+
+## 破坏性变更
+
+从 beta 版到发布 rc 版，我们有几个重大变化。这些可以自动迁移(见上文)，也可以手动执行。
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx]:
```suggestion
从 Beta 版到发布 RC 版，我们有几个重大变化。这些可以自动迁移（见上文），也可以手动执行。
```
Formatting nitpick.

File: src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+### Tauri 核心插件
+
+我们在功能中改变了Tauri内置插件的处理方式 [PR #10390](https://github.com/tauri-apps/tauri/pull/10390).
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx]:
```suggestion
我们在功能中改变了 Tauri 内置插件的处理方式 [PR #10390](https://github.com/tauri-apps/tauri/pull/10390)。
```
Formatting nitpick.

File: src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+要从最新的 beta 版本迁移，您需要在您的能力中添加所有核心权限标识符 `core:`，或者切换到 `core:default` 权限并删除旧的核心插件标识符。
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx]:
```suggestion
要从最新的 Bbeta 版本迁移，您需要在您的能力中添加所有核心权限标识符 `core:`，或者切换到 `core:default` 权限并删除旧的核心插件标识符。
```
Formatting nitpick.

File: src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+...
+"permissions": [
+    "core:default"
+]
+...
+```
+
+### 内置开发服务器
+
+我们对内置开发服务器的网络公开进行了更改 [PR #10437](https://github.com/tauri-apps/tauri/pull/10437) and [PR #10456](https://github.com/tauri-apps/tauri/pull/10456).
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx]:
```suggestion
我们对内置开发服务器的网络公开在 [PR #10437](https://github.com/tauri-apps/tauri/pull/10437) 和 [PR #10456](https://github.com/tauri-apps/tauri/pull/10456) 进行了更改。
```
Wording fixes.

File: src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+内置的移动开发服务器不再公开网络范围，并且将本地机器上的流量直接隧道传输到设备。
+
+目前，在 iOS 设备上运行时（无论是直接运行还是通过 Xcode），此改进不会自动应用。
+在这种情况下，我们默认使用开发服务器的公共网络地址，但有一种方法可以绕过这个问题，即打开Xcode以自动建立您的macOS机器与连接的iOS设备之间的连接，
+然后运行 `tauri ios dev --force-ip-prompt` 来选择iOS设备的TUN地址（以 **::2** 结尾）。
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx]:
```suggestion
在这种情况下，我们默认使用开发服务器的公共网络地址，但有一种方法可以绕过这个问题，即打开 Xcode 以自动建立您的 macOS 机器与连接的 iOS 设备之间的连接，然后运行 `tauri ios dev --force-ip-prompt` 来选择 iOS 设备的 TUN 地址（以 **::2** 结尾）。
```
Formatting nitpick.

File: src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+
+如果打算在物理 iOS 设备上运行，您的开发服务器配置需要适应此更改。
+以前我们建议检查 `TAURI_ENV_PLATFORM` 环境变量是否与 `android` 或 `ios` 匹配，
+但是现在除非使用 iOS 设备，否则我们可以连接到本地主机，您应该检查 `TAURI_DEV_HOST` 环境变量。
+以下是一个 Vite 配置迁移的示例。
+
+- 2.0.0-beta:
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx]:
```suggestion
- 2.0.0-beta：
```
Formatting nitpick.

File: src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+          protocol: 'ws',
+          host: internalIpV4Sync(),
+          port: 1421,
+        }
+      : undefined,
+  },
+});
+```
+
+- 2.0.0-rc:
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx]:
```suggestion
- 2.0.0-rc：
```
Formatting nitpick.

**Review by dklassic — CHANGES_REQUESTED** [review]
Some formatting nitpicks here and there and we should be good to go!

File: src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx (hunk 4 above)
Comment (GuoJikun) [reply src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx]: > Formatting nitpick.

Bbeta  ->  Beta

Comment (GuoJikun) [reply src/content/docs/zh-cn/start/migrate/from-tauri-2-beta.mdx]:
```suggestion
要从最新的 Beta 版本迁移，您需要在您的能力中添加所有核心权限标识符 `core:`，或者切换到 `core:default` 权限并删除旧的核心插件标识符。
```

**Pull request merged**